src/index.ts

```diff
@@ -1,5 +1,6 @@
-import provider from "./components/provider";
+import provider from './components/provider';
 
+// eslint-disable-next-line import/no-anonymous-default-export
 export default {
```

Comment:
Rather than ignore this lint error, you could make this a named export instead:

```suggestion
export {
```

I think it's equivalent when `esModuleInterop` is enabled.